[2024-09-03T10:41:32.387+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: k9_etl_dag_v3.check_updates manual__2024-09-03T10:41:26.125122+00:00 [queued]>
[2024-09-03T10:41:32.399+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: k9_etl_dag_v3.check_updates manual__2024-09-03T10:41:26.125122+00:00 [queued]>
[2024-09-03T10:41:32.400+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-09-03T10:41:32.415+0000] {taskinstance.py:1380} INFO - Executing <Task(_PythonDecoratedOperator): check_updates> on 2024-09-03 10:41:26.125122+00:00
[2024-09-03T10:41:32.421+0000] {standard_task_runner.py:57} INFO - Started process 219 to run task
[2024-09-03T10:41:32.423+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'k9_etl_dag_v3', 'check_updates', 'manual__2024-09-03T10:41:26.125122+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/k9_etl_dag_v3.py', '--cfg-path', '/tmp/tmpzpytzvko']
[2024-09-03T10:41:32.427+0000] {standard_task_runner.py:85} INFO - Job 16: Subtask check_updates
[2024-09-03T10:41:32.471+0000] {task_command.py:415} INFO - Running <TaskInstance: k9_etl_dag_v3.check_updates manual__2024-09-03T10:41:26.125122+00:00 [running]> on host 2624bebdd8ee
[2024-09-03T10:41:32.562+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='k9_etl_dag_v3' AIRFLOW_CTX_TASK_ID='check_updates' AIRFLOW_CTX_EXECUTION_DATE='2024-09-03T10:41:26.125122+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-03T10:41:26.125122+00:00'
[2024-09-03T10:41:32.600+0000] {python.py:194} INFO - Done. Returned value was: No new facts found.
[2024-09-03T10:41:32.621+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=k9_etl_dag_v3, task_id=check_updates, execution_date=20240903T104126, start_date=20240903T104132, end_date=20240903T104132
[2024-09-03T10:41:32.697+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-09-03T10:41:32.735+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
